server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v3.2.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-fix-api-error-messages",
```

Comment:
Note to self, update this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/14 is merged.

Reply:
Done.